### PR TITLE
#2101 - Student Application Updates: Introduce grant-only for PART-TIME capability

### PR DESCRIPTION
**Pull request by ann-aot**
- Added a new question in the student application, which will appear when a student selects `part-time` 
image
- When the option `loans and grants` is selected below info banner will appear,
image
- Option ` Grants Only` will not have any banners in the UI.
- When ` Grants Only` is selected, the student is assessed for only grant funding. Assessment is performed normally and added a logic in the workflow to check the award type options and if it's only `grants`, then it will zero out the loan.
- Part-time has only Canada loan, so, we are only making Canada loan `0` when `grants only` is selected.
image

- Assessment and when `loans and grants` are selected
- 1 disbusement
image
image
- 2 disbursements
image
image

- Assessment and NOA when grants only` are selected
- 1 disbusement
image
image
- 2 disbursements
image
image

- Fixed an info banner issue in the program tab
image

### CODE REVIEW

File: sources/packages/backend/workflow/BPMN/camunda-8/load-assessment-consolidated-data.bpmn [hunk 1]
```diff
@@ -99,6 +99,7 @@
           <zeebe:header key="studentDataStudentParentNetAssests" value="$.data.studentParentNetAssests" />
           <zeebe:header key="studentDataStudentParentNetContribution" value="$.data.studentParentNetContribution" />
           <zeebe:header key="studentDataPartnerTotalIncomeAssistance" value="$.data.partnertotalincomeAssistance" />
+          <zeebe:header key="studentDataPartTimeAwardTypesToBeConsidered" value="$.data.partTimeAwardTypes" />
```
Comment (andrewsignori-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/load-assessment-consolidated-data.bpmn:102]: I believe that we try to follow the patter where the `studentData` is juts added as a prefix without changing the variable name itself. Can we keep this, please?

It should be either
- studentDataPartTimeAwardTypesToBeConsidered
- partTimeAwardTypesToBeConsidered
or
- studentDataPartTimeAwardTypes
- partTimeAwardTypes

**Review by andrewsignori-aot — COMMENTED** [review]
Great work, it looks good. Just a minor comment about variable naming.

File: sources/packages/forms/src/form-definitions/sfaa2023-24.json [hunk 2]
```diff
@@ -1983,7 +1984,7 @@
                       "refreshOnChange": false,
                       "customClass": "banner-info",
                       "key": "html12",
-                      "customConditional": "show = data.programnotListed || data.offeringnotListed;",
+                      "customConditional": "show = data.myProgramNotListed.programnotListed || data.myStudyPeriodIsntListed.offeringnotListed;",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/sfaa2023-24.json:1987]: What is this change about ?

Comment (dheepak-aot) [reply sources/packages/forms/src/form-definitions/sfaa2023-24.json:1987]: Do we need a null check here ? `data.myProgramNotListed?.programnotListed`

Comment (ann-aot) [reply sources/packages/forms/src/form-definitions/sfaa2023-24.json:1987]: image

**Review by dheepak-aot — APPROVED** [review]
Thanks for the change. Great Job overall. 👍

**Review by sh16011993 — APPROVED** [review]
Nice work @ann-aot 👍

**Review by andrepestana-aot — APPROVED** [review]
Looks good.

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/sfaa2021-22.json:1]: Why this file is way more updated than others?

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for accommodating the last-minute change in the requirement. Looks good 👍

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @ann-aot

**Pull request merged**